### PR TITLE
Updated React Tutorial in Gitbooks to remove errors generated in Create React App

### PR DESCRIPTION
**Pull request by tgrrr**

### CODE REVIEW

File: docs/tutorials.md [hunk 1]
```diff
@@ -168,7 +167,7 @@ your own plugins. Plugins have two parts, one plugin definition and a
 ReactJS component. A minimal plugin definition looks as followed
 
 ```jsx
-import React from 'react'
+import React, {Component} from 'react'
```
Comment (arekkas) [inline docs/tutorials.md]: I don't think the component is required here

File: docs/tutorials.md [hunk 2]
```diff
@@ -118,7 +117,7 @@ export default {
 and a minimalistic plugin example could look like:
 
 ```jsx
-import React from 'react'
+import React, {Component} from 'react'
```
Comment (arekkas) [inline docs/tutorials.md]: I don't think the component is required here

File: docs/tutorials.md [hunk 3]
```diff
@@ -97,7 +96,7 @@ your own plugins. Plugins have two parts, one plugin definition and a
 ReactJS component. A minimal plugin definition looks as followed
 
 ```jsx
-import React from 'react'
+import React, {Component} from 'react'
```
Comment (arekkas) [inline docs/tutorials.md]: I don't think the component is required here

**Comment by tgrrr** [comment]
Agreed. [Updated here.](https://github.com/ory/editor/pull/500/commits/549af99a79a7725ddaa50e89800000fdef437a1f)

File: docs/tutorials.md (hunk 3 above)
Comment (arekkas) [reply docs/tutorials.md]: @tgrrr I think you missed this line :)

**Pull request merged**